libsnark/zk_proof_systems/plonk/tests/bls12_381_test_vector_transcript_hasher.hpp

```diff
@@ -0,0 +1,125 @@
+/** @file
+ *****************************************************************************
+ * @author     This file is part of libff, developed by Clearmatics Ltd
+ *             (originally developed by SCIPR Lab) and contributors
+ *             (see AUTHORS).
+ * @copyright  MIT license (see LICENSE file)
+ *****************************************************************************/
+
+#ifndef LIBSNARK_ZK_PROOF_SYSTEMS_PLONK_TESTS_BLS12_381_TEST_VECTOR_TRANSCRIPT_HASHER_HPP_
+#define LIBSNARK_ZK_PROOF_SYSTEMS_PLONK_TESTS_BLS12_381_TEST_VECTOR_TRANSCRIPT_HASHER_HPP_
+
+namespace libsnark
+{
+
+/// transcript hasher interface specific to the BLS12-381 curve
```

Comment:
Small nit.  Something like:
```
/// Implementation of the transcript hasher interface (see transcript_hasher.hpp), returning hard-coded test vector values for the plonk proof system on a specific circuit, using the BLS12-381 curve.
```

Reply:
`Also performs a simple check on the transcript to be hashed to verify correct operation of the plonk prover and verifier.`

Reply:
Done.